packages/opentelemetry-sdk-trace-base/src/export/SimpleSpanProcessor.ts

```diff
@@ -45,7 +44,7 @@ export class SimpleSpanProcessor implements SpanProcessor {
   }
 
   // does nothing.
-  onStart(_span: Span, _parentContext: Context): void {}
```

Comment:
I guess `BatchSpanProcessorBase.ts` needs a similar change

Reply:
and probably also `NoopSpanProcessor`